### DISCUSSION TITLE
Show release notes after update

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#833] [Ideas]
### What feature would you like to see?

Currently, after updating, Zen shows https://www.zen-browser.app/welcome

It would be nice to change that to https://www.zen-browser.app/release-notes/latest

### What platform are you seeing the problem on?

Linux

### Additional context

EndeavourOS w/ 6.10.2 kernel, `zen-browser-bin` installed & updated using `yay`
 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/446_

**Comment by vnwr** [comment]
I would prefer the option for neither to be shown. I don't like the way the page currently forces itself open and into focus when the browser launches. I'd rather not be disrupted like that by a page that I'm just going to close right away.

**Reply by Aya-Komichi** [reply]
I second this. I would rather get the option to not have my web browser automatically open tabs without my consent.

And if the devs really really believe we should read all the release notes, please ask us first with a prompt at the very least? The web itself already lacks consent for many things, at least let me have that with the browser I use.

And web browsers in general are updated frequently, having the release notes forced into my sight every time isn't practical at all. The only web browser I ever read release notes from was firefox and only because I'm slightly doubtful about what Mozilla adds to it these days (they haven't been doing that well after all).
